packages/manager/modules/vps/src/routing.js

```diff
@@ -27,11 +33,22 @@ export default /* @ngInject */ ($stateProvider) => {
       resolve: {
         connectedUser: /* @ngInject */ (OvhApiMe) =>
           OvhApiMe.v6().get().$promise,
-        capabilities: /* @ngInject */ (serviceName, OvhApiVpsCapabilities) =>
+        capabilities: /* @ngInject */ (
+          coreConfig,
+          isVpsNewRange,
+          serviceName,
+          OvhApiVpsCapabilities,
+        ) =>
           OvhApiVpsCapabilities.Aapi()
             .query({ serviceName })
             .$promise.then((capabilities) =>
-              capabilities.map((capability) => kebabCase(capability)),
+              capabilities
+                .filter((capability) =>
+                  !isVpsNewRange && coreConfig.isRegion('US')
+                    ? capability === DASHBOARD_FEATURES.rebuild
```

Comment:
Why aren't capabilities only handled by the `vps-capabilities` 2api ?

Reply:
I choose that the `vps-capabilities` just returns all items for US, and then it is filtered here (as it is meant to be removed). Also it is to avoid to send to 2api serviceName + modelName (which contains information of vps is new range or not)

Reply:
No I meant that the 2API could tell you when a capability is available for a new range or a legacy, but don't change anything